install_config/install/host_preparation.adoc

```diff
@@ -72,6 +72,20 @@ packages.
     --enable="rhel-7-server-extras-rpms" \
     --enable="rhel-7-server-ose-3.4-rpms"
 ----
++
+[NOTE]
+====
+If the `subscription-manager repos --disable="*"` command does not disable all
+repositories, use `yum-config-manager --disablerepo=<repo_name>`. If the
+repositories are not disabled then enabled in the method above, your system
```

Comment:
The part that says "not disabled then enabled in the method above" is unclear / confusing to me.
What does "method above" refer to, the four `subscription-manager ...` commands in the Host Registration section, or something else?
"not disabled then enabled", I didn't understand that.